dev-python/iniparse/iniparse-0.4-r2.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Better INI parser for Python"
+HOMEPAGE="https://code.google.com/p/iniparse https://pypi.python.org/pypi/iniparse"
+SRC_URI="https://${PN}.googlecode.com/files/${P}.tar.gz"
```

Comment:
HOMEPAGE is still up, but download link not anymore. Not sure if the one from pypi.python.org would be better though: https://pypi.python.org/packages/0f/d1/3090ef9be165da5ddb1b0cf2b332d3282588bdd2dd0967e94b547f10055f/iniparse-0.4.tar.gz